src/components/accordion/_accordion.scss

```
@@ -1,3 +1,13 @@
+@mixin iconHightlight {
+  @include euiFocusRing;
+  color: $euiColorPrimary;
+}
+
+@mixin iconMarginAlignRight {
+  margin-left: $euiSizeS;
+  margin-right: $euiSizeXS;
+}
```

A best practice we utilize is to keep mixins separated into their own file. You'll want to create a file called `_mixins.scss` as a sibling to this one and import that file into the `_index.scss` file before the import of this one. The index file will then look something like 

```scss
@import 'mixins';
@import 'accordion';
@import 'accordion_form';
```

---

We also make sure we prefix everything with `eui`, even component-specific mixins. So I would rename these to:

```suggestion
@mixin euiAccordionIconFocus {
  @include euiFocusRing;
  color: $euiColorPrimary;
}

@mixin euiAccordionIconMargin {
  margin-left: $euiSizeS;
  margin-right: $euiSizeXS;
}
```

---

@cchaos For the `euiAccordionIconMargin` mixin, do you think it would be better to add a condition `right` with default value to `false` since the name is general?

```scss
@mixin euiAccordionIconMargin($right: false) {
    @if $right {
        margin-left: $euiSizeS;
        margin-right: $euiSizeXS;
    } @else {
         margin-left: $euiSizeXS;
         margin-right: $euiSizeS;
     }
}
```

---

Sure!